rust-peer/src/main.rs

```diff
@@ -199,6 +224,39 @@ async fn main() -> Result<()> {
                 SwarmEvent::Behaviour(BehaviourEvent::Kademlia(e)) => {
                     debug!("Kademlia event: {:?}", e);
                 }
+                SwarmEvent::Behaviour(BehaviourEvent::RequestResponse(
+                    request_response::Event::Message { message, .. },
+                )) => match message {
+                    request_response::Message::Request { request, .. } => {
+                        todo!("request_response::Message::Request: {:?}", request);
```

Comment:
You've specified `ProtocolSupport::Full` below so this may very well happen. If you don't want to allow inbound requests, you should specify a different `ProtocolSupport`.

Reply:
Good point: 0f46513